test/controllers/sessionsController.test.js

```diff
@@ -0,0 +1,82 @@
+const request = require('supertest');
+const app = require('../../app');
+
+describe('POST #signip', () => {
+  beforeEach(async done => {
+    await request(app)
```

Comment:
instead of this use something like factory girl and faker to create db entries instead